### DISCUSSION TITLE
A Miller based GitHub's Flat Data action example

### DISCUSSION THREAD

**Discussion by aborruso** [johnkerl/miller#526] [Show and tell]
Two days ago it was announced [**Flat Data**](https://octo.github.com/projects/flat-data): it explores how to make it easy to work with data in git and GitHub.

It uses GitHub action to archive automatically data on a repo (git scraping). Moreover you can apply some post processing job.
The official way to post process a file is to use Deno; I have created an hello world example that uses Milller (via bash)  https://github.com/aborruso/flat_data_bash_example

**Comment by aborruso** [comment]
The example repo is now in the official Flat postprocessing documentation https://github.com/githubocto/flat-postprocessing#using-bash

**Reply by johnkerl** [reply]
@aborruso I will definitely be checking this out -- thanks for the tip!!! :D
